server/lease/lessor_test.go

```diff
@@ -150,11 +150,19 @@ func TestLeaseConcurrentKeys(t *testing.T) {
 	wg.Wait()
 }
 
+func TestLessorRevoke(t *testing.T) {
+	lessorRevoke(t, true)
+}
+
+func TestLessorRevokeWithoutRangeDeleter(t *testing.T) {
```

Comment:
Problem with adding this test is fact that in any correct configuration `RangeDeleter` should only be set. `RangeDeleter` can be nil when creating Lessor and set later just to avoid circular dependencies in bootstap and allow changing reference to backend during recovery. We should rather work to make the

Reply:
Yes, this is a valid comment. I rolled back the change for the test case for now. Let's address the task of `removing circular dependencies` in separate PR if needed.

Reply:
SG